numba/cuda/cudadrv/driver.py

```diff
@@ -2670,12 +2661,25 @@ def complete(self):
         """
 
 
+def _import_mvc_packages():
+    try:
+        from ptxcompiler import compile_ptx
+        from cubinlinker import CubinLinker, CubinLinkerError
```

Comment:
I think this is importing these into the local scope of `_import_mvc_packages()`, rather than the `driver` module?

Reply:
Oh, that's possible. I was still in the mindset of the other fix for cuDF where we were patching the module directly.

We have two options:
1. Declare the variables as `global` like this simplified example -- the `global` scope is the module scope here:
```
def _import_mvc_packages():
    global compile_ptx
    from ptx_compiler import compile_ptx
```
2. Import the names more precisely, only in the functions that require them.

Reply:
I'd prefer option 2, because it keeps the effects of the import more localised, rather than stashing something into a module global when it's only used in one other place anyway.

Reply:
Fantastic. I agree that's best and I will implement those changes. Probably tomorrow.

Reply:
Resolved in b062fee.